pallets/creditcoin/src/lib.rs

```diff
@@ -794,7 +794,6 @@ pub mod pallet {
 			};
 
 			Self::use_guid(&guid)?;
```

Comment:
I think the purpose of `use_guid` was to avoid re-using the same guid as a key for the offchain index. If we still want to do the check and not allow a shared guid between bid and ask orders, maybe we should move the check up with the other `ensure` statements.